Sadot/User/UserHomePage.cs

```diff
@@ -123,12 +195,21 @@ private void OpenOrderForm(Table tmpTable)
 
             if (orderForm.DialogResult == System.Windows.Forms.DialogResult.OK)
             {
-                db.UpdateTableStatus(tmpTable.TableID, "תפוס");
+                db.UpdateTableParams(tmpTable.TableID, "תפוס", "הזמנה בהכנה", DateTime.Now);
             }
             if (orderForm.DialogResult == System.Windows.Forms.DialogResult.Yes)
```

Comment:
else is missing before the if, no? (else if), or use switch/case

Reply:
Fixed
I have rewritten the function - review it again, please